.gitignore

```diff
@@ -1,9 +1,4 @@
 /pushgateway
-dependencies-stamp
-/.build
```

Comment:
Please keep `/.build`, `/.release` and `/.tarballs`, they are used by the release process.